types/next-pwa/index.d.ts

```diff
@@ -0,0 +1,262 @@
+// Type definitions for next-pwa 5.6
+// Project: https://github.com/shadowwalker/next-pwa#readme
+// Definitions by: Nivaldo Farias <https://github.com/NivaldoFarias>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.6
+
+/// <reference types="react"/>
```

Comment:
Is this reference required?

Reply:
Apparently not!! Guess I unknowingly used it as a workaround while trying to fix the mismatch to _DOM_ declarations issue [mentioned above](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63391#issuecomment-1328129756).

Reply:
Apparently it does!! jadshgdasj minutes after I sent the previous reply, the CI failed. 

Refer to [these error logs](https://dev.azure.com/definitelytyped/DefinitelyTyped/_build/results?buildId=144987&view=logs&j=8635df7d-ecde-52a7-c7a8-84b998e35690&t=c264d665-0b79-59b7-7e04-9965cbe74860&l=129). Is it a common issue? If so, is there a default solution or should I re-introduce the reference to `@types/react`?

Reply:
Please. Add the React typings into package.json.

Reply:
Awaiting [PR merge](https://github.com/microsoft/DefinitelyTyped-tools/pull/571) on `allowedPackageJsonDependencies.txt` in order to push the changes to the package _(remove `@types/react` triple-dash reference in favour of adding to package.json as a dependency)_